migrations/2024-02-29-130650_complete-metadata-creation/up.sql

```diff
@@ -0,0 +1,6 @@
+alter table nft_metadata
```

Comment:
Are any of these planned to be not null?

Reply:
Yes, mb_internal_id should not be null for Mintbase v2 metadata, all the others are still optional but might not be null.